en/docs/learn/working-with-acr-and-amr.md

```diff
@@ -274,6 +274,49 @@ Follow the steps below to try the scenario without using an application:
 	
 	Note that the AMR values in this example are 	`DemoFaceIdAuthenticator`	, 	`BasicAuthenticator`	, and 	`DemoFingerprintAuthenticator`. These are the authenticators that are used in the authentication process.
 
+#### Translating AMR values
+Follow the steps below to translate the AMR values that get returned to the authentication process.
+
+1. Open the `deployment.toml` file in the `<IS_HOME>/repository/conf directory`.
+
+2. Add the following configurations
+
+    !!! note 
+    You can add any custom AMR mapping to this config.
```

Comment:
Does this render properly when you build the doc locally?
I believe it should be aligned as:

!!! note
    You can add

Reply:
Fix with a5c790700af9084435520e134ba8f71368868655
<img width="711" alt="Screenshot 2022-04-28 at 9 49 48 PM" src="https://user-images.githubusercontent.com/32576163/165799771-b2948a0e-3598-43c9-8b2c-94a3f903deb5.png">